openshift/app.dc.yaml

```diff
@@ -258,3 +276,17 @@ parameters:
     description: Requested Memory per pod (in gigabytes Gi or megabytes Mi ex. 500Mi)
     displayName: Memory Request
     value: 256Mi
+  - name: FILE_CACHE_DIR
```

Comment:
should probably rename all this from FILE_CACHE_DIR, that was for cdogs/carbone because the library that required the pvc was file-cache.

Reply:
I've gone ahead and changed it to DATA_*. Also looked up unix mounting locations and changed default to /mnt/app/data instead as it more closely matches conventions.